api/database/seeders/DatabaseSeeder.php

```diff
@@ -21,6 +21,7 @@ public function run()
         $this->call(CmoAssetSeeder::class);
         $this->call(OperationalRequirementSeeder::class);
         $this->call(DepartmentSeeder::class);
+        $this->call(PoolCandidateSearchRequestSeeder::class);
```

Comment:
So for, we've been using the Seeder classes when we want to create models with specific values. For generating random models, we can just include the factory method in this file, like with Users and Pools. Also, it should go _after_ Pools are generated, so that they can link to Pools.